frontend/src/components/pages/CompleteProfilePage.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import Select from "react-select";
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  Input,
+  Heading,
+  IconButton,
+  Text,
+  Textarea,
+} from "@chakra-ui/react";
+import { Icon } from "@chakra-ui/icon";
+import { MdFolder } from "react-icons/md";
+import Header from "../navigation/Header";
+
+const DropdownIndicator = () => {
```

Comment:
this looks to be repeated from `TableFilter` - mind making a file for this component in `components/utils` and importing the component in `TableFilter` and `CompleteProfilePage?`